doc/source/whatsnew/v0.18.0.txt

```diff
@@ -136,6 +137,78 @@ New Behavior:
    s.index
    s.index.nbytes
 
+.. _whatsnew_0180.enhancements.extract:
+
+Changes to str.extract
+^^^^^^^^^^^^^^^^^^^^^^
+
+The :ref:`.str.extract <text.extract>` method takes a regular
+expression with capture groups, finds the first match in each subject
+string, and returns the contents of the capture groups.
+
+In v0.18.0, the ``expand`` argument was added to ``extract``. When
+``expand=False`` it returns a ``Series``, ``Index``, or ``DataFrame``,
+depending on the subject and regular expression pattern (same behavior
+as pre-0.18.0). When ``expand=True`` it always returns a
+``DataFrame``, which is more consistent and less confusing from the
```

Comment:
give a little explanation of the requirement for passing ``expand`` (otherwise you will get the warning)